public/app/features/admin/UserListPage.tsx

```diff
@@ -47,6 +47,7 @@ export default function UserListPage() {
   const hasAccessToOrgUsers = contextSrv.hasPermission(AccessControlAction.OrgUsersRead);
   const hasEmailSharingEnabled =
     Boolean(config.featureToggles.publicDashboards) &&
+    config.publicDashboardsEnabled &&
```

Comment:
should we abstract this piece of code into a new function in utils such as "isPublicDashboardsEnabled"? 
```
const isPublicDashboardsEnabled = () => {
  return Boolean(config.featureToggles.publicDashboards) && 
   config.publicDashboardsEnabled
}
```

Reply:
I updated the PR!